payments_config/__init__.py

```diff
@@ -62,6 +64,10 @@ def __init__(self, seller, config):
             raise ValueError('Recurrence should be not set or monthly: {}'.
                              format(self.recurrence))
 
+        if self.user_identification not in self.valid_user_identifications:
+            raise ValueError('user_identification must be one of: {}'.
+                             format(str(self.valid_user_identifications)))
```

Comment:
oh right, that's annoying. It would have to be `', '.join(repr(v) for v in self.valid_user_identifications)`